# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: ./package/.../ooo/hotfix_always_unpack_crystal.patch
# Copyright (C) 2004 - 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
always unpack crystal icons because cws-iconswitching1.diff needs that
Alejandro

--- bin/unpack.orig	2006-01-28 23:24:39.000000000 -0300
+++ bin/unpack	2006-01-28 23:25:07.000000000 -0300
@@ -284,9 +284,9 @@
 # fix a checkbox bug - in the up-stream icon set.
     rm -f $CUSTOM_DIR/industrial/vcl/source/src/check*; 
 fi
-if test "z${WITH_ICONS#*crystal}" != "z$WITH_ICONS" ; then
+#if test "z${WITH_ICONS#*crystal}" != "z$WITH_ICONS" ; then
     tar xjf $SRCDIR/$OOO_CRYSTAL_IMAGES -C $CUSTOM_DIR || exit 1;
-fi
+#fi
 
 if test "x$OOO_EXTRA_ARTWORK" != "x"; then
 # Html export rulers etc.
